caddytls/config.go

```diff
@@ -269,6 +270,11 @@ func MakeTLSConfig(configs []*Config) (*tls.Config, error) {
 	}
 
 	return &tls.Config{
+		// A tls.Config must have Certificates or GetCertificate
+		// set, in order to be accepted by tls.Listen and quic.Listen.
+		GetCertificate: func(*tls.ClientHelloInfo) (*tls.Certificate, error) {
```

Comment:
Can you add a TODO here so that we remember to remove this after it is safe to do so?